www/_posts/2023-05-22-cordova-android-12.0.0.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: post
+author:
+    name: Bryan Ellis
+title:  "Cordova Android 12.0.0 Released!"
+categories: announcements
+tags: news releases
+---
+
+We are happy to announce that we have just released `Cordova Android 12.0.0`! This is one of Cordova's supported platforms for building Android applications.
+
+* [cordova-android@12.0.0](https://www.npmjs.com/package/cordova-android)
+
+## Release Highlights
+
+**To upgrade:**
+
+```bash
+cordova platform remove android
+cordova platform add android@12.0.0
+```
+
+**To install:**
+
+```bash
+cordova platform add android@12.0.0
+```
+
+* **Increased Minimum & Target SDK**
+
+    This release has increased the minimum supported SDK version to 24 which is Android 7.0. It also has increased the target SDK to 33, Android 13.
+
+* **Tooling and Default Support Bump**
+
+  * Target SDK (`targetSdk`): `33`
+  * SDK Build Tool: `33.0.2`
+  * Kotlin: `1.7.21`
+  * Gradle: `7.6`
+  * Android Gradle Plugin (AGP): `7.4.2`
+  * Google Services Gradle Plugin: `4.3.15`
+  * AndroidX App Compat Library: `1.6.1`
+  * AndroidX WebKit Library: `1.6.0`
+  * AndroidX SplashScreen Core Library: `1.0.0`
+
+* **Monochrome Support**
+
+    Android 13 has added Themed Icons support which is also known as Monochrome. This release has introduced support for Themed icons.
+
+* **Node Support**
+
+    We have dropped support for Node 14.x and increase the minimum Node requirement to greater than or equal to 16.13.0.
+
+Please report any issues you find at [issues.cordova.io](http://issues.cordova.io/)!
+
+<!--more-->
+# Changes include:
+
+**Breaking:**
+
```

Comment:
I think here we can add a bit more...

Perhaps explicitly mention the requirement of installing build tools 33.0.2.

Below I'l include a potential candidate...

### START OF SNIPPET

```
## Build Tools

To use cordova-android@12, build tools 33.0.2 must be installed. Older build tools version can be uninstalled if older versions of cordova-android is no longer used in your projects.

To install build tools 32.0.2:

1. open Android Studio's SDK manager:
2. Click on `SDK Tools` tab
3. Check `Show Package Details`
4. Expand `Android SDK Build-Tools`
5. Check `33.0.2`
6. Click `Apply`
```

![Screenshot from 2023-05-22 12-01-35](https://github.com/apache/cordova-docs/assets/11200662/08db78fb-c2e2-40ca-bd40-8d5cffd3e5fa)

### END OF SNIPPET

I think the bullet point list of breaking change can be left as is. This is just a recommendation to explicitly mention build tools as this will be required to use cordova-android@12.